site/_docs/history.md

```diff
@@ -27,24 +27,120 @@ For a full list of releases, see
 <a href="https://github.com/apache/calcite/releases">github</a>.
 Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
-
 <!--
-## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.35.0">1.35.0</a> / XXXX-XX-XX
-{: #v1-35-0}
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.36.0">1.36.0</a> / XXXX-XX-XX
+{: #v1-36-0}
 
-This release comes x months after [1.34.0](#v1-34-0),
+This release comes x months after [1.35.0](#v1-35-0),
 contains contributions from x contributors, and resolves x issues.
 
 Contributors to this release:
 x,
 y (release manager),
 z.
 
+#### Breaking Changes
+{: #breaking-1-36-0}
+
+Compatibility: This release is tested on Linux, macOS, Microsoft Windows;
+using JDK/OpenJDK versions 8 to 19;
+Guava versions 16.0.1 to 31.1-jre;
+other software versions as specified in gradle.properties.
+
+#### New features
+{: #new-features-1-36-0}
+
+#### Dependency version upgrade
+{: #dependency-1-36-0}
+
+#### Bug-fixes, API changes and minor enhancements
+{: #fixes-1-36-0}
+
+#### Build and test suite
+{: #build-1-36-0}
+
+#### Web site and documentation
+{: #site-1-36-0}
+-->
+
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.35.0">1.35.0</a> / 2023-07-13
+{: #v1-35-0}
+
+This release comes 4 months after [1.34.0](#v1-34-0),
+contains contributions from 36 contributors, and resolves 139 issues.
+
+Among other new features, we have added extensive function support. For example, we added roughly 14 new functions in BigQuery library for handling array, dates, math, and string.
+Also, we added 27 new functions in Spark library for handling array, map and about 11 new functions for other database products.
+
+Among other improvements and fixes, it is worth highlighting the following improvements:
+
+* Some improvements in calcite core.
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5703">CALCITE-5703</a>]
+    Reduce amount of generated runtime code
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5479">CALCITE-5479</a>]
+    FamilyOperandTypeChecker is not readily composable in sequences
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5425">CALCITE-5425</a>]
+    Should not pushdown Filter through Aggregate without group keys
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5506">CALCITE-5506</a>]
+    RelToSqlConverter should retain the aggregation logic when Project without RexInputRef on the Aggregate
+
+* Some improvements in simplify an expression.
+  * [<a href="https://issues.apache.org/jira/browse/CALCITE-5769">CALCITE-5769</a>]
+    Optimizing 'CAST(e AS t) IS NOT NULL' to 'e IS NOT NULL'
```

Comment:
Instead of single quotes use backticks (`) for SQL keywords and functions. Lots of class names and SQL functions/keywords below are not properly formatted so this comment applies to the whole section.

Reply:
Done.